src/util/screening-selector.js

```diff
@@ -36,11 +55,11 @@ export async function getScreeningInstrument(client) {
     ).then((module) => module.default);
     return [questionnaireNidaQs, elmJsonNidaQs, valueSetJson];
   } else {
-    if (!client) throw new Error("invalid FHIR client provided");
     let libId = screeningInstrument.toUpperCase();
     // load questionnaire from FHIR server
     const qSearch = await client
-      .request("/Questionnaire?name="+screeningInstrument)
+      // look up the questionnaire based on whether the id or the name attribute matches the specified instrument id?
+      .request("/Questionnaire?_id|name:contains="+screeningInstrument)
```

Comment:
Using name as a secondary is interesting, will keep an eye out for others doing this